sections/testingandquality/aaa.md

```diff
@@ -1,20 +1,23 @@
-# Structure tests by the AAA pattern

```

Comment:
Hey, thanks for the contribution! I see you mistakenly changed the English version instead of the Korean one :)